src/sidebar/components/sidebar-content-error.js

```diff
@@ -7,9 +7,14 @@ import Button from './button';
 import SvgIcon from '../../shared/components/svg-icon';
 
 /**
- * An error message to display in the sidebar.
+ * Show an error indicating that an annotation or group referenced in the URL
+ * could not be fetched.
  */
-export default function SidebarContentError({ errorType, onLoginRequest }) {
+export default function SidebarContentError({
+  errorType,
+  onLoginRequest,
+  showClearSelection = false,
```

Comment:
Did you have a reason for making the default `false`?

Reply:
By popular vote! - We had three use cases for this error in the app and it was false for two of them.